### PR TITLE
[Security] New cmdlets for Settings API + Updated Nuget package

### PR DESCRIPTION
**Pull request by taklei**
## Description

The design review - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/597

- Added new cmdlets for Settings API 
- Updated Nuget package of Security
- Added tests & help files
- Rerun & fixed broken tests of JIT API & Compliance API

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Security/Security/help/Set-AzSecuritySetting.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+### Microsoft.Azure.Commands.Security.Models.Settings.PSSecuritySetting
+### Microsoft.Azure.Commands.Security.Models.Settings.PSSecurityDataExportSetting
+
+
+### System.Boolean
+
+## OUTPUTS
+
+### Microsoft.Azure.Commands.Security.Models.Settings.PSSecuritySetting
```
Comment (VeryEarly) [inline src/Security/Security/help/Set-AzSecuritySetting.md:168]: should be only one output type PSSecuritySetting

File: src/Security/Security/help/Set-AzSecuritySetting.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
```
Comment (VeryEarly) [inline src/Security/Security/help/Set-AzSecuritySetting.md:158]: input should also have string

File: src/Security/Security/help/Get-AzSecuritySetting.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```
Comment (VeryEarly) [inline src/Security/Security/help/Get-AzSecuritySetting.md:4]: please add the online version

**Comment by taklei** [comment]
@VeryEarly fixed everything :)

**Comment by VeryEarly** [comment]
LGTM

**Pull request merged**